features/filesystem/fat/FATFileSystem.cpp

```diff
@@ -205,8 +206,8 @@ DRESULT disk_read(BYTE pdrv, BYTE *buff, DWORD sector, UINT count)
 {
```

Comment:
Could the disk functions could still live in the mbed namespace if you declare them as `extern "C"`?

Reply:
If they are declared as extern "C" then they are part of the global namespace; the only one C knows.

Reply:
Though declaring them as extern "C" allows you to use anything from the mbed namespace and also notates why the functions aren't in the namespace.

It's just a minor thing to note